pymatgen/entries/tests/test_compatibility.py

```diff
@@ -632,10 +731,102 @@ def test_ozonide(self):
                                                                         {'titel': 'PAW_PBE O 08Apr2002',
                                                                          'hash': '7a25bc5b9a5393f46600a4939d357982'}]})
 
-        lio3_entry_corrected = self.compat.process_entry(lio3_entry)
+        lio3_entry_corrected = self.compat._process_entry(lio3_entry)
         self.assertAlmostEqual(lio3_entry_corrected.energy, -3.0 - 3 * 0.66975)
 
 
+class TestMaterialsProjectAqueousCompatibility():
+    """
+    Test MaterialsProjectAqueousCompatibility
+
+    -x- formation energy of H2O should always be -2.458 eV/H2O
+    -x- H2 energy should always be the same value
+    -x- H2O energy should always be the same value
+    -x- Should get warnings if you init without all energy args
+    -x- Should get CompatibilityError if you get_entry without all energy args
+    -x- energy args should auto-populate from entries passed to process_entries
+    -x- check compound entropies appropriately added
+    -x- check hydrate adjustment appropriately applied
+
+    Notes:
+        Argument values from MaterialsProjectCompatibility as of April 2020:
+            corrected DFT energy of H2O = -15.5875 eV/H2O (mp-697111) or -5.195 eV/atom
+            corrected DFT energy of O2 = -4.9276 eV/atom (mp-12957)
+            total energy corrections applied to H2O (eV/H2O) -0.70229 eV/H2O or -0.234 eV/atom
+    """
+    def test_MPAqeous_H_H2O_energy_with_args(self):
```

Comment:
typo (also pep violation).. should've started a review rather than individual comments, apologies

Reply:
What exactly is the PEP violation in `test_MPAqeous_H_H2O_energy_with_args`? are function names not supposed to have capitals?

Reply:
how about `test_h_h2o_energy_with_args`?